<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>txSectUDP</class>
 <widget class="QMainWindow" name="txSectUDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP Transmission</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="IPaddr">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="portNo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmDetails">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm IP/Port</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>301</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="textEnterUDP"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="orgDatagram"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="timedTxUDP">
           <property name="text">
            <string>Send UDP (2Hz)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendUDP">
           <property name="text">
            <string>Send UDP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearUDP">
           <property name="text">
            <string>Clear UDP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="timedTxUDPstop">
         <property name="text">
          <string>Stop Timed Sending</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readDisp">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Read/Display</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inDisp">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>72</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outDisp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>104</y>
      <width>271</width>
      <height>239</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
